api/v2.go

```diff
@@ -235,18 +233,12 @@ func (c *V2Context) GetMultisigAlias(w web.ResponseWriter, r *web.Request) {
 	}()
 
 	ownerAddr := r.PathParams["owner"]
-	// convert owner address from bech32 to be used internally
-	addr, err := address.ParseToID(ownerAddr)
-	if err != nil {
-		c.WriteErr(w, 400, err)
-		return
-	}
 
 	c.WriteCacheable(w, caching.Cacheable{
 		TTL: 5 * time.Second,
-		Key: c.cacheKeyForID("multisig_alias", addr.String()),
+		Key: c.cacheKeyForID("multisig_alias", ownerAddr),
```

Comment:
Using bech32 here makes the cache only useable for 1 chain now.
Is there any reason for moving address parsing to GetMSigAlias?